h/static/scripts/tests/controllers/search-bar-controller-test.js

```diff
@@ -23,9 +23,11 @@ describe('SearchBarController', function () {
   var dropdown;
   var dropdownItems;
   var ctrl;
+  var form;
 
   before(function () {
-    template = '<input data-ref="searchBarInput">' +
+    template = '<form>' +
```

Comment:
Protip: You can use template literals (the JS equivalent of Python's `"""strings"""`) as an easier way to write multi-line strings:

```js
var template = `<form>
...
</form>`
```